wgsl/index.bs

```diff
@@ -1011,9 +1011,13 @@ An attribute [=shader-creation error|must not=] be specified more than once per
         [=shader-creation error|Must=] be positive.
     <td>[=shader-creation error|Must=] only be applied to a member of a [=structure=] type.
 
-    [=shader-creation error|Must=] be a power of 2, and [=shader-creation error|must=] satisfy the required-alignment for the member type:
+    [=shader-creation error|Must=] be a power of 2.
 
-    <p algorithm="align constraint">
+    Note: This attribute influences how a value of the enclosing structure type can appear in memory:
+    at which byte addresses the structure itself and its component members can appear.
+    In particular, the rules in [[#memory-layouts]] combine to imply the following constraint:
+
+    <p class="note" algorithm="implied constraint on align attribute">
```

Comment:
Hey editors: Should that "must" on line 1024 be a link to shader-creation error? It's not an independent check.

Reply:
Unless it breaks the logic, I think linking musts to there is convenient